security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -59,10 +58,11 @@ type Plugin struct {
 	// map key is token type, map value is tokenInfo.
 	tokens           sync.Map
 	gCPProjectNumber string
+	gKEClusterURL    string
```

Comment:
should be `gkeClusterURL` . This does not seem to follow golang naming best practice, by changing the casing in the middle of an acronym)
Same comment for `gCPProjectNumber` above that should be `gcpProjectNumber`

Reply:
All of the naming fixes are in https://github.com/istio/istio/pull/20823.